content/en/platform/corda/4.9/enterprise/release-notes-enterprise.md

```diff
@@ -135,7 +135,7 @@ In this release:
 
 * Corda shell has been removed to its own repository for improved security. You can now use a standalone shell outside of the node, or from within the node's drivers. For more information about using the standalone shell, see [The standalone shell](./node/operating/shell.html#the-standalone-shell). For information on adding the shell to the node's drivers, see [Upgrading a node to Corda Enterprise Edition 4.9](node-upgrade-notes.html).
 {{< note >}} 
-The Corda shell, which has a dependency on Groovy, is a standalone product since Corda Enterprise Edition 4.9. As a result, for any flows dependent on Groovy, you must add the dependency to the CorDapp or the drivers' directory.
+For any flows dependent on Groovy, you must add the dependency to the CorDapp or the drivers' directory.
```

Comment:
I think now you're just missing the context of why they need add the dependency. So something like:

The Corda shell has a dependency on Groovy that the Corda API does not. As a result, for any flows dependent on Groovy, you must now add the dependency to the CorDapp or the drivers' directory.

Reply:
Updated!